tests/helpers/testers.py

```diff
@@ -136,7 +142,8 @@ def _class_test(
 
     total_preds = torch.cat([preds[i] for i in range(NUM_BATCHES)])
     total_target = torch.cat([target[i] for i in range(NUM_BATCHES)])
-    sk_result = sk_metric(total_preds, total_target)
+    total_kwargs_update = {k: torch.cat([v[i] for i in range(NUM_BATCHES)]) for k, v in kwargs_update.items()}
```

Comment:
same as above

Reply:
update in https://github.com/PyTorchLightning/metrics/pull/137/commits/86e4015cb0bf85bd10c7f860ada0c8a0c93124d5